### PR TITLE
Update web-content-filtering.md

### PR DESCRIPTION
**Pull request by msbemba**
Added Network protection and Defender smart screen to pre requisites 

Also added network protection requirements to related topics

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/5052

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/security/defender-endpoint/web-content-filtering.md [hunk 1]
```diff
@@ -57,6 +57,7 @@ Before trying out this feature, make sure you have the following requirements:
 - Windows 10 Enterprise E5, Microsoft 365 E5, Microsoft 365 E5 Security, Microsoft 365 E3 + Microsoft 365 E5 Security add-on or the Microsoft Defender for Endpoint standalone license. 
 - Access to Microsoft Defender Security Center portal (https://securitycenter.windows.com).
 - Devices running Windows 10 Anniversary Update (version 1607) or later with the latest MoCAMP update.
+- Windwos Defender Smartscreen and Network protection enabled.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/web-content-filtering.md]:
```suggestion
- Windows Defender SmartScreen and Network protection enabled.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Dansimp Please merge this Pr. Thanks

**Pull request merged**